static/app/components/profiling/flamegraph/flamegraphSpans.tsx

```diff
@@ -120,13 +137,31 @@ export function FlamegraphSpans({
       );
     };
 
-    drawSpans();
+    const drawText = () => {
+      spansTextRenderer.draw(
+        spansView.configView.transformRect(spansView.configSpaceTransform),
+        spansView.fromTransformedConfigView(spansCanvas.physicalSpace)
+        // flamegraphSearch.results
```

Comment:
I assume this is copy pasted?

Reply:
Yeah, these are all the same textRenderer calls :(